sdk/storage/Azure.Storage.Blobs/tests/ServiceClientTests.cs

```diff
@@ -752,7 +752,7 @@ public async Task UndeleteBlobContainerAsync_Error()
         }
 
         [Test]
-        [PlaybackOnly("https://github.com/Azure/azure-sdk-for-net/issues/18257")]
+        [Ignore("https://github.com/Azure/azure-sdk-for-net/issues/18257")]
```

Comment:
what's the plan to bring back this test?

Reply:
We will bring back Container Rename tests when the feature is enabled in the service.